src/relations/backend_database.py

```diff
@@ -73,6 +78,7 @@ def _on_database_created(self, event: DatabaseCreatedEvent) -> None:
 
         Accesses user and password generated by the postgres charm and adds a user.
         """
+        logger.info("this hook is definitely firing.... ")
         try:
             cfg = self.charm.read_pgb_config()
```

Comment:
do you also need to defer on PathError or is it better to error out? (I'm a newb here on pgb)